### PR TITLE
Feature/launch icons

### PR DESCRIPTION
**Pull request by SERDUN**
- rename:  logo to primary_onboardin_logo
- added:    secondary_onboardin_logo (for use on different color layouts)

### CODE REVIEW

File: assets/launcher_icons/android.png
Comment (ycherniavskyi) [inline assets/launcher_icons/android.png:1]: Could you please clarify why added asset has a bit strange resolution: W: 415px | H: 415px?

File: assets/launcher_icons/ios.png
Comment (ycherniavskyi) [inline assets/launcher_icons/ios.png:1]: The deleted asset was intentionally created with a resolution of W: 1024px | H: 1024px in order to enhance the quality of the assets generated based on it.

**Review by ycherniavskyi — CHANGES_REQUESTED** [review]
Is it possible to align resolution of modified assets? If resolution must be changed then please provide links to docs that clarify it.

**Comment by SERDUN** [comment]
Android 13:
adaptive icons
1 layer: image: color [108x108]
2 layer: image: color [108x108] ilusration [72x72]+ (better add 16 padding)

Android <13:
icons  image: color [432:432]  for max resolution drawable-xxxhdpi

**Review by ycherniavskyi — APPROVED** [review]

**Pull request merged**